Orange/widgets/data/owselectcolumns.py

```diff
@@ -361,6 +361,12 @@ def __init__(self):
                           Orange.data.ContinuousVariable))
 
         self.used_attrs_view.setModel(self.used_attrs)
+
+        ua = self.used_attrs
+        ua.dataChanged.connect(self.update_completer_model)
+        ua.rowsInserted.connect(self.update_completer_model)
+        ua.rowsRemoved.connect(self.update_completer_model)
```

Comment:
The update_completer_model is probably not what you want to call here. It only updates the completion model for the filter line edit in the left "Available Variables" box. It does not trigger an update of the widget's output.
Note also that changes in the `class_attrs` and `meta_attrs` should also trigger an update. Note also that a drag/drop between the used_attrs/class/meta_attrs_views will trigger possible multiple rowsRemoved signals from drag source model followed by a rowsInserted signal in the destination model. Ideally the widget output should only be updated/computed once, after the drag/drop operation has been completed.